### PR TITLE
Add style gem

### PR DESCRIPTION
**Pull request by camirmas**
Notes:
- I called it `blue_steel` but we can rename to something more appropriate if necessary 👔 
- It does not alter the default task for the app that installs it. Maybe add a script that will do that after the fact

### CODE REVIEW

File: ruby/blue_steel/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+you one criterion to consider when you want to propose a change:
+
+1. The style is probably not going to change, and that's ok!
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'blue_steel'
```
Comment (tcollier) [inline ruby/blue_steel/README.md:24]: Will this be a public gem that is published to ruby gems? If not, then the gem installation will need to point to the URL.

File: ruby/blue_steel/Rakefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require 'bundler/gem_tasks'
+load 'lib/tasks/style.rake'
+task default: %i[style:branch]
+
+task :environment do
```
Comment (tcollier) [inline ruby/blue_steel/Rakefile:5]: This doesn't seem to do anything. You probably want to remove it or am I missing something?

**Review by tcollier — APPROVED** [review]

File: ruby/blue_steel/Rakefile (hunk 2 above)
Comment (camirmas) [reply ruby/blue_steel/Rakefile:5]: without it I get an error saying `:environment` task cannot be run

File: ruby/blue_steel/README.md (hunk 1 above)
Comment (camirmas) [reply ruby/blue_steel/README.md:24]: I think we were planning to publish it but I'd defer to @kristjan on that one

**Review by jish — APPROVED** [review]

**Pull request merged**